src/components/Header/HeaderMain.jsx

```diff
@@ -153,9 +153,11 @@ export default function HeaderMain() {
 	}));
 	const dispatch = useDispatch();
 	const user = JSON.parse(localStorage.getItem("user"));
-
 	const navigate = useNavigate();
 
+	const handleRedirectToWishlist = () => {
+		  window.location.href = "/user/wishlist";
+	};
```

Comment:
Window.location reload the app, this kills the intention of React [SPA]
Please make sure to navigate of useNavigate() from react router